rules/eslint/errors/on.js

```diff
@@ -28,8 +28,8 @@ module.exports = {
     "no-ex-assign": 2,
     // disallow double-negation boolean casts in a boolean context
     "no-extra-boolean-cast": 2,
-    // disallow unnecessary parentheses
-    "no-extra-parens": 2,
+    // allow unnecessary parentheses
```

Comment:
NIT: I wouldn't change the comment. The rule is to "disallow". You're just turning it off with `0`.